pkg/scheduler/framework/v1alpha1/framework.go

```diff
@@ -604,3 +641,13 @@ func (f *framework) pluginsNeeded(plugins *config.Plugins) map[string]config.Plu
 	}
 	return pgMap
 }
+
+// status is a double pointer to Status because Status is not created when arguments
+// of this function is evaluated in "defer".
+func recordExtensionPointDuration(start time.Time, extensionPoint string, status **Status) {
```

Comment:
Why doulbe pointer is necessary?

Reply:
because the arguments to deferred functions are evaluated immediately. Without the double pointer, it will always be nil.

Reply:
Refactored the code to avoid using double pointer.